runtime/android/core/strings/xwalk_app_strings.grd

```diff
@@ -2,62 +2,59 @@
 <grit current_release="1" latest_public_release="0">
   <release allow_pseudo="false" seq="1">
     <messages fallback_to_english="true">
+      <message desc="Crosswalk Install Title [CHAR-LIMIT=100]" name="IDS_CROSSWALK_INSTALL_TITLE">
+        Crosswalk Project Install
+      </message>
       <message desc="Decompression Progress Message [CHAR-LIMIT=100]" name="IDS_DECOMPRESSION_PROGRESS_MESSAGE">
-        Decompressing Crosswalk Library
+        Decompressing Crosswalk Project runtime...
       </message>
       <message desc="Download Progress Message [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_PROGRESS_MESSAGE">
-        Downloading Crosswalk
-      </message>
-      <message desc="Download Failed Title [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_TITLE">
-        Crosswalk Download Failed
+        Downloading Crosswalk Project from remote host...
       </message>
       <message desc="Download Failed Message [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_MESSAGE">
-        Please retry or cancel
+        Download failed. A network error occurred.
       </message>
       <message desc="Download Failed Not Found [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_DEVICE_NOT_FOUND">
-        Storage device not found
+        Download failed. Storage device not found.
       </message>
       <message desc="Download Failed Insufficient space [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_INSUFFICIENT_SPACE">
-        Insufficient storage space
+        Download failed. Insufficient disk space.
       </message>
       <message desc="Download Failed Time Out [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_TIME_OUT">
-        Timed out
-      </message>
-      <message desc="Market Open Failed Title [CHAR-LIMIT=100]" name="IDS_MARKET_OPEN_FAILED_TITLE">
-        Market Open Failed
+        Download failed. The network connection timed out.
       </message>
       <message desc="Market Open Failed Message [CHAR-LIMIT=100]" name="IDS_MARKET_OPEN_FAILED_MESSAGE">
-        Please set up your market
+        Failed to open the app store
```

Comment:
Decide if you want "." at the end of errors and be consistent on all.  Also, for this error, is there anything the user can do?  The error doesn't help them solve it.   Something like:  "Failed to open the app store.  Please setup your app store account."  (just a guess)